guides/source/active_storage_overview.md

```diff
@@ -1415,6 +1415,33 @@ end
 [fixtures]: testing.html#the-low-down-on-fixtures
 [`ActiveStorage::FixtureSet`]: https://api.rubyonrails.org/classes/ActiveStorage/FixtureSet.html
 
+### Configuring services
+
+You can add `config/storage/test.yml` to configure services to be used in test environment
+This is useful when the `service` option is used.
+
+```ruby
+class User < ApplicationRecord
+  has_one_attached :avatar, service: :s3
+end
+```
+
+Without `config/storage/test.yml`, the `s3` service configured in
+`config/storage.yml` is used - even when running tests. 
+The default configuration would be used and files would be uploaded to the service provider configured in `config/storage.yml`.
+In this case, you can add `config/storage/test.yml` and use Disk service for
+`s3` service to prevent sending requests.
```

Comment:
```suggestion
Without `config/storage/test.yml`, the `s3` service configured in `config/storage.yml` is used - even when running tests.

The default configuration would be used and files would be uploaded to the service provider configured in `config/storage.yml`.

In this case, you can add `config/storage/test.yml` and use Disk service for `s3` service to prevent sending requests.
```

Reply:
This should either be one paragraph, ideally wrapped around 80 columns, or 3 separate ones -- whichever fits best.